frontend/ui/data/dictionaries.lua

```diff
@@ -122,6 +122,1592 @@ local dictionaries = {
         license = _("GNU/FDL"),
         url = "http://http.debian.net/debian/pool/non-free/s/stardict-german-czech/stardict-german-czech_20161201.orig.tar.gz",
     },
+    -- Dictionaries mirrored from Sourceforge, see https://github.com/koreader/koreader/pull/3176#issuecomment-447085441
+    {
+        name = "Afrikaans-English dictionary",
+        lang_in = "Afrikaans",
+        lang_out = "English",
+        entries = 4198,
+        license = "GNU Public License",
+        url = "https://gitlab.com/avsej/dicts-stardict-form-xdxf/raw/264aadf8/002c/stardict-comn_sdict05_afrikaans-english-2.4.2.tar.bz2",
+    },
+    {
+        name = "Albanian-English dictionary",
```

Comment:
We should probably change all of these to something like `FREELANG Lang A – Lang B dictionary` for clarity.

Reply:
I was looking for the FREELANG license and it's not permissive. I'll send them an e-mail, but these may have to be excluded.